src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -320,7 +404,7 @@ private void verifyExceptionRecordIsNotAttachedToCase(
 
         Object attachToCaseRef = fetchedExceptionRecord.getData().get(ATTACH_TO_CASE_REFERENCE);
 
-        if (attachToCaseRef != null && Strings.isNotEmpty(attachToCaseRef.toString())) {
+        if (attachToCaseRef != null && !Strings.isNullOrEmpty(attachToCaseRef.toString())) {
```

Comment:
I don't think this change will help - if `attachToCaseRef` is null, an exception will still be thrown.